src/iscn/iscnId.test.ts

```diff
@@ -0,0 +1,53 @@
+import { ISCNRecordData } from '../types';
+import { getMsgCreateIscnRecordJSON, getISCNIdPrefix } from './iscnId';
+
+const dummyMsgCreateIscnRecord :{
+  sender: string;
+  iscnRecord: ISCNRecordData;
+} = {
+  sender: 'like1ukmjl5s6pnw2txkvz2hd2n0f6dulw34h9rw5zn',
+  iscnRecord: {
+    recordNotes: '',
+    stakeholders: [],
+    contentFingerprints: [],
+    contentMetadata: {},
```

Comment:
Adding some content to some of the fields would be better

Reply:
I didn't write that...